kolibri/core/analytics/tasks.py

```diff
@@ -8,7 +9,7 @@
 from kolibri.core.analytics.utils import DEFAULT_SERVER_URL
 from kolibri.core.analytics.utils import ping_once
 from kolibri.core.tasks.decorators import register_task
-from kolibri.core.tasks.exceptions import JobRunning
+from kolibri.core.tasks.exceptions import JobRunning, JobNotRunning
```

Comment:
Lint lint lint!

Reply:
I'm not seeing what you mean here

Reply:
This is breaking the linting, because we enforce one import per line - pre-commit should fix this.